# Polymorphism.Interfaces

makarenko09 opened the pull request:
execute taskThree();

Commit:
execute taskThree

Commit:
fix this[Только один момент при печати - sout экземпляра сроки Article из метода search класса Searchable не использует перевод на новую строку в терминале:

         [Искусственный интеллект в медицинеИскусственный интеллект (ИИ) революционизирует сферу здравоохранения, предоставляя новые инструменты для диагностики, лечения и прогнозирования заболеваний. Данная статья рассматривает текущие применения ИИ в медицине и потенциальные перспективы его развития., null, null, null, null]

         /*@Override
         public String toString() {
             return name + title;
         }
         */] ->    @Override
                   public String toString() {
                       return name +"\n"+ title;
                   }

kalyashov commented on src/org/skypro/skyshop/Searchable.java:
```diff
@@ -0,0 +1,19 @@
+package org.skypro.skyshop;
```
перенеси в отдельный пакет org.skypro.skyshop.search

kalyashov commented on src/org/skypro/skyshop/SearchEngine.java:
```diff
@@ -0,0 +1,31 @@
+package org.skypro.skyshop;
```
перенеси в отдельный пакет org.skypro.skyshop.search

kalyashov commented on src/org/skypro/skyshop/Searchable.java:
```diff
@@ -0,0 +1,19 @@
+
+public interface Searchable {
+
+    String searchTerm(String request);
```
В этом интерфейсе должно быть 3 метода и методы не должны ничего принимать на вход по условиям задания, обсуждали сегодня на разборе ДЗ

Commit:
fix all-other

         [Искусственный интеллект в медицинеИскусственный интеллект (ИИ) революционизирует сферу здравоохранения, предоставляя новые инструменты для диагностики, лечения и прогнозирования заболеваний. Данная статья рассматривает текущие применения ИИ в медицине и потенциальные перспективы его развития., null, null, null, null]

         /*@Override
         public String toString() {
             return name + title;
         }
         */] ->    @Override
                   public String toString() {
                       return name +"\n"+ title;
                   }

Commit:
rewrite method of Interface

Commit:
!!!rewrite method of Interface
